doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -87,42 +98,41 @@ The following client-side profile is based on the rules that sets schema.
   "build" : "1"
 }
 ```
-The schema structure from our example allows filtering the endpoints by the operation systems of device (for example, to show only Android devices), os_version (for example, to push some 
-notifications only for the specified os version). It is allowed to create complex filtering conditions by combining as many filtering conditions as needed.
 
-### Setting client-side endpoint profile schema from Admin UI
+The schema structure from our example allows filtering the endpoints by the operation system of device (for example, to show only Android devices), os_version (for example, to push some notifications only for the specified os version).
+You can create complex filtering conditions by combining as many filtering conditions as needed.
 
-The list of client-side endpoint profile schemas created by a tenant developer for the application is shown in the **Client-side EP profile** schemas window, which can 
-be opened from the application menu on the navigation panel as illustrated by the following screenshot.
+### Setting client-side EP profile schema from Administration UI
+
+To view the list of client-side endpoint profile schemas created by a [tenant developer]({{root_url}}Glossary/#tenant-developer), open the the **Client-side EP profile** page under the **Schemas** section of the application.
```

Comment:
You should not talk about EP profile schemas here. Rather, your EP profile references CTL schemas.